recipes/r-memuse/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: r-memuse
+  version: "3.0_1"
+
+source:
+  fn: memuse_3.0-1.tar.gz
```

Comment:
Can we move this to the conda-forge channel?
This will help you to create recipes for conda-forge: https://github.com/bgruening/conda_r_skeleton_helper

Reply:
Sure @bgruening. Should I push these new recipes to bioconda or conda-forge now ?

Reply:
All non-bio related packages should be deposit into conda-forge. This would be great!

Reply:
Thank you @bgruening . I removed the r-memuse recipe from this branch, and pushed it to conda-forge.